packages/cactus-plugin-ledger-connector-besu/README.md

```diff
@@ -99,6 +99,125 @@ enum Web3SigningCredentialType {
 ```
 > Extensive documentation and examples in the [readthedocs](https://readthedocs.org/projects/hyperledger-cactus/) (WIP) 
 
+
+### Building/running the container image locally
+
+In the Cactus project root say:
+
+```sh
+DOCKER_BUILDKIT=1 docker build -f ./packages/cactus-plugin-ledger-connector-besu/Dockerfile . -t cplcb
+```
+
+Build with a specific version of the npm package:
+```sh
+DOCKER_BUILDKIT=1 docker build --build-arg NPM_PKG_VERSION=0.4.1 -f ./packages/cactus-plugin-ledger-connector-besu/Dockerfile . -t cplcb
+```
+
+#### Running the container
+
+Launch container with plugin configuration as an **environment variable**:
+```sh
+docker run \
+  --rm \
+  --publish 3000:3000 \
+  --publish 4000:4000 \
+  --env PLUGINS='[{"packageName": "@hyperledger/cactus-plugin-ledger-connector-besu", "type": "org.hyperledger.cactus.plugin_import_type.LOCAL",  "options": {"rpcApiHttpHost": "http://localhost:8545", "instanceId": "some-unique-besu-connector-instance-id"}}]' \
+  cplcb
+```
+
+Launch container with plugin configuration as a **CLI argument**:
+```sh
+docker run \
+  --rm \
+  --publish 3000:3000 \
+   --publish 4000:4000 \
+  cplcb \
+    ./node_modules/.bin/cactusapi \
+    --plugins='[{"packageName": "@hyperledger/cactus-plugin-ledger-connector-besu", "type": "org.hyperledger.cactus.plugin_import_type.LOCAL",  "options": {"rpcApiHttpHost": "http://localhost:8545", "instanceId": "some-unique-besu-connector-instance-id"}}]'
+```
+
+Launch container with **configuration file** mounted from host machine:
+```sh
+
+echo '[{"packageName": "@hyperledger/cactus-plugin-ledger-connector-besu", "type": "org.hyperledger.cactus.plugin_import_type.LOCAL",  "options": {"rpcApiHttpHost": "http://localhost:8545", "instanceId": "some-unique-besu-connector-instance-id"}}]' > cactus.json
+
+docker run \
+  --rm \
+  --publish 3000:3000 \
+  --publish 4000:4000 \
+  --mount type=bind,source="$(pwd)"/cactus.json,target=/cactus.json \
+  cplcb \
+    ./node_modules/.bin/cactusapi \
+    --config-file=/cactus.json
+```
+
+#### Testing API calls with the container
+
+Don't have a Besu network on hand to test with? Test or develop against our Besu All-In-One container!
+
+**Terminal Window 1 (Ledger)**
+```sh
+docker run -p 0.0.0.0:8545:8545/tcp  -p 0.0.0.0:8546:8546/tcp  -p 0.0.0.0:8888:8888/tcp  -p 0.0.0.0:9001:9001/tcp  -p 0.0.0.0:9545:9545/tcp hyperledger/cactus-besu-all-in-one:latest
+```
+
+**Terminal Window 2 (Cactus API Server)**
+```sh
+docker run \
+  --network host \
+  --rm \
+  --publish 3000:3000 \
+  --publish 4000:4000 \
+  --env PLUGINS='[{"packageName": "@hyperledger/cactus-plugin-ledger-connector-besu", "type": "org.hyperledger.cactus.plugin_import_type.LOCAL",  "options": {"rpcApiHttpHost": "http://localhost:8545", "instanceId": "some-unique-besu-connector-instance-id"}}]' \
```

Comment:
@jordigiam Is it not needed to add here the keychain memory plugin? I was just guessing that it is based on you having included the npm install of it in the image. What I'm trying to make sure is that these example commands (all of them) still work with the updates to the image you just made, if you can confirm that then I'm good.

Reply:
Without keychain memory is working, but only with the main version of the plugins:

<img width="1042" alt="Captura de pantalla 2021-04-22 a las 11 30 37" src="https://user-images.githubusercontent.com/78140052/115691263-2f416400-a35e-11eb-8074-0d583af9b099.png">

Reply:
Understood! That's good enough, we should publish v0.5.0 soon then so this problem fixes itself when the fresh package versions hit the @latest dist-tag on npm. Thanks for double checking this @jordigiam